core/schema/modulesource.go

```diff
@@ -1977,7 +1982,7 @@ func (s *moduleSourceSchema) moduleSourceGeneratedContextDirectory(
 func (s *moduleSourceSchema) runModuleDefInSDK(ctx context.Context, src, srcInstContentHashed dagql.Instance[*core.ModuleSource], mod *core.Module) (*core.Module, error) {
 	runtimeImpl, ok := src.Self.SDKImpl.AsRuntime()
 	if !ok {
-		return nil, fmt.Errorf("sdk implementation does not implement Runtime interface")
+		return nil, fmt.Errorf("the SDK does not implement the `Runtime` interface")
```

Comment:
We can create an error type for this as well, even though we don't handle it today.

It may be useful to handle it in the future.

Reply:
Done